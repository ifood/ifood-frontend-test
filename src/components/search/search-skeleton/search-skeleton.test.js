import React from 'react'
import { render } from '@testing-library/react'
import { SearchSkeleton } from '.'

describe('SearchSkeleton', () => {
  let component

  beforeEach(() => {
    component = render(<SearchSkeleton />)
  })

  it('should render component correctly', () => {
    const { asFragment } = component
    expect(asFragment()).toMatchSnapshot()
  })
})
