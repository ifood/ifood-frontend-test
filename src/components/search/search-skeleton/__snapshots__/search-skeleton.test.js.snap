// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchSkeleton should render component correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="MuiSkeleton-root MuiSkeleton-rect makeStyles-skeleton-1 MuiSkeleton-animate"
    />
    <div
      class="MuiSkeleton-root MuiSkeleton-rect makeStyles-skeleton-1 MuiSkeleton-animate"
    />
    <div
      class="MuiSkeleton-root MuiSkeleton-rect makeStyles-skeleton-1 MuiSkeleton-animate"
    />
    <div
      class="MuiSkeleton-root MuiSkeleton-rect makeStyles-skeleton-1 MuiSkeleton-animate"
    />
    <div
      class="MuiSkeleton-root MuiSkeleton-rect makeStyles-skeleton-1 MuiSkeleton-animate"
    />
  </div>
</DocumentFragment>
`;
